applicants/README.md

```diff
@@ -0,0 +1,154 @@
+# applicants
+
+## applicants object
+
+```{
+  userId: string,
+  biodata: {
+    firstName: string,
+    lastName: string,
+  },
+  location: {
+    city: string,
+    state: string,
+    country: string,
+  },
+  professional: {
+    institution: string,
+    skills: string,
+  },
+  intro: {
+    introduction: string,
+    funFact: string,
+    forFun: string,
+    whyRds: string,
+    numberOfHours: string,
+  },
+  foundFrom: string,
+  status: string || null,
+  discordLink: string || null
+}
+```
+
+## **Requests**
+
+|                         Route                          |             Description              |
+| :----------------------------------------------------: | :----------------------------------: |
+|                [GET /applications](#get-applications)                |   Returns all the applications in the system   |
+|           [GET /applications/:userId](#get-applicationsuserid)            | Retruns application of a particular user |
+|  [POST /applications](#post-applications)   |    Add application of a user    |
+|       [PATCH /applications/:userId](#patch-applicationsuserid)       |   Updates application of a user   |
+|        |
+
+## **GET /applications/**
+
+Return all the applications which are not accepted or rejected, this API will only be accessible to super_user
+
+- **Params**  
+  None
+- **Query** 
+  None
+- **Body**  
+  None
+- **Headers**  
+  Content-Type: application/json
+- **Cookie**  
+  rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+    `{ 'message': 'applications returned successfully!', 'applications': <application_object> }`
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
+  - **Code:** 500
+    - **Content:**
+      `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'An internal server error occurred' }`
+
+## **POST /applications/**
+
+Adds an application for joining RDS, a user can only add one application, until the previous application is either rejected or accepted by super_user
```

Comment:
what about handling/responses after rejection?

Reply:
add logs for previous rejections